tests/conftest.py

```diff
@@ -98,13 +107,15 @@ def qc():
 
 @pytest.fixture
 def dataset(request):
-    if request.param == "htqc":
-        return htqc()
+    dataset_param = request.param
 
-    if request.param == "cellpainting":
-        return cellpainting()
+    if dataset_param == "htqc":
+        return request.getfixturevalue('htqc')
```

Comment:
Could you please add a comment as to why we need to use `getfixturevalue`?

Reply:
Good point - I will add!